ui/tutorial-renderer/src/main/res/layout/tutorial_youtube_activity.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <data>
+        <variable name="activity" type="org.cru.godtools.tutorial.activity.YoutubePlayerActivity" />
+        <variable name="lifecycleOwner2" type="androidx.lifecycle.LifecycleOwner" />
+        <variable name="videoId" type="String" />
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:onClick="@{() -> activity.finish()}">
```

Comment:
I haven't seen this UI but imagine based on this that there are two ways to close the activity and know that the user is done watching the onboarding video:

1. the on ```onPlaybackEnded``` function
2. This ```onClick``` here. Is this like an X up in the corner or something?

Reply:
yeah, just 2 different ways to dismiss the video. The UI is actually a modal, this `onClick` captures and clicks outside of the video itself